corehq/apps/case_search/xpath_functions.py

```diff
@@ -35,6 +38,32 @@ def date(node):
     )
 
 
-XPATH_FUNCTIONS = {
+def _selected_query(node, fuzzy, operator):
```

Comment:
nit: this function should after the other functions to follow [Newspaper Metaphor](https://github.com/jnguyen095/clean-code/issues/28)